.github/workflows/release.yml

```
@@ -0,0 +1,76 @@
+name: Manual Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to release'
+        required: true
+    branches:
```

actionlint (1.7.4)

expected "inputs" key for "workflow_dispatch" section but got "branches"

(syntax-check)